packages/utils/src/constants.ts

```diff
@@ -43,7 +43,7 @@ export const HTML_TAGS_REGEX = /<[^>]*>/g;
 export const HTML_REGEX = /<([A-Za-z][A-Za-z0-9]*)\b[^>]*>(.*?)<\/\1>/;
 export const COUPON_REGEX = /[a-zA-Z0-9!@#$%^&*()_+=-}{?.,]/;
 export const EMAIL_REGEX = /^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/;
-export const USER_NAME_REGEX = /^[\u0600-\u06ff\s]+$/;
+export const USER_NAME_REGEX = /^[(A-Za-z|\u0600-\u06ff)\s]+$/;
```

Comment:
I took some time to think about it and I come up with these rules: 

1. Student can have their names in Arabic or English.
2. Tutors much be in Arabic. 

This why we should do the following: 

```suggestion
// All users expect of tutors and tutors managers can have their names in either Arabic or English or both. 
// https://regex101.com/r/ER7McY/1
export const USER_NAME_REGEX = /^[A-Za-z\u0600-\u06ff ]+$/; 
// Tutor/tutor mangers name must be in Arabic - https://regex101.com/r/Av5SGH/1
export const TUTOR_NAME_REGEX = /^[\u0600-\u06ff\s]+$/;; 
```